BraveWallet/Crypto/Stores/NetworkSelectionStore.swift

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import BraveCore
+import BraveShared
+import SwiftUI
+
+class NetworkSelectionStore: ObservableObject {
+  
+  struct NetworkPresentation: Equatable, Hashable, Identifiable {
+    var id: String { network.id }
+    let network: BraveWallet.NetworkInfo
+    let subNetworks: [BraveWallet.NetworkInfo]
+    let isPrimaryNetwork: Bool
+    
+    init(
+      network: BraveWallet.NetworkInfo,
+      subNetworks: [BraveWallet.NetworkInfo],
+      isPrimaryNetwork: Bool
+    ) {
+      self.network = network
+      self.subNetworks = subNetworks
+      self.isPrimaryNetwork = isPrimaryNetwork
+    }
+  }
+  
+  var networkStore: NetworkStore
+  
+  @Published var primaryNetworks: [NetworkPresentation] = []
+  @Published var secondaryNetworks: [NetworkPresentation] = []
```

Comment:
These can be `private(set)` I assume?

Reply:
Fixed in [c29f314](https://github.com/brave/brave-ios/pull/5706/commits/c29f314f45ae50559626c245a8c40fe2dbdfd34e)